### PR TITLE
feat(OffersPage.js): add info events for offer detail [MIDUWEB-1537]

### PR DESCRIPTION
**Pull request by schmidt1024**
Figma: https://jira.migros.net/browse/MIDUWEB-1537

Demo (localhost):
1 http://localhost:3000/src/es/components/web-components-toolbox/docs/TemplateMiduweb.html?rootFolder=src&css=./src/css/variablesCustomKlubschule.css&login=./src/es/components/molecules/login/default-/default-.html&logo=./src/es/components/atoms/logo/default-/default-.html&nav=./src/es/components/web-components-toolbox/src/es/components/molecules/multiLevelNavigation/default-/default-.html&footer=./src/es/components/organisms/footer/default-/default-.html&content=./src/es/components/pages/Infoveranstaltungen.html
2 http://localhost:3000/src/es/components/web-components-toolbox/docs/TemplateMiduweb.html?rootFolder=src&css=./src/css/variablesCustomKlubschule.css&login=./src/es/components/molecules/login/default-/default-.html&logo=./src/es/components/atoms/logo/default-/default-.html&nav=./src/es/components/web-components-toolbox/src/es/components/molecules/multiLevelNavigation/default-/default-.html&footer=./src/es/components/organisms/footer/default-/default-.html&content=./src/es/components/pages/generator/https---www-klubschule-ch-gesundheit-sport-und-fitnesskurse-kurs-bauch-beine-po-online--D_91461_2687_1442-center-2678.html

### CODE REVIEW

**Comment by schmidt1024** [comment]
PR is on hold until BE delivers some courses (info events) to test.

File: src/es/components/controllers/withFacet/WithFacet.js [hunk 1]
```diff
@@ -54,9 +54,11 @@ export default class WithFacet extends WebWorker() {
     this.url = new URL(self.location.href)
     this.params = this.catchURLParams()
     const isMocked = this.hasAttribute('mock')
-    const endpoint = isMocked
+    const isMockedInfoEvents = this.hasAttribute('mock-info-events')    
+    let endpoint = isMocked
```
Comment (Weedshaker) [inline src/es/components/controllers/withFacet/WithFacet.js:58]: shortHand + if = mixed salad 🥰

File: src/es/components/molecules/tile/Tile.js [hunk 2]
```diff
@@ -349,7 +349,7 @@ export default class Tile extends Shadow() {
         </div>
         <div class="m-tile__foot">
           <div class="m-tile__foot-left">
-            ${this.hasAttribute('is-wish-list') && !this.isPassed ? /* html */`<a-icon-mdx namespace="icon-mdx-ks-" icon-name="Trash" size="1em" request-event-name="remove-from-wish-list" course="${data.parentkey ? `${data.parentkey}${data.centerid ? `_${data.centerid}` : ''}` : `${data.kurs_typ}_${data.kurs_id}_${data.centerid}`}"></a-icon-mdx>` : ''}
+            ${this.hasAttribute('is-wish-list') && !this.isPassed && !this.hasAttribute('is-info-events') ? /* html */`<a-icon-mdx namespace="icon-mdx-ks-" icon-name="Trash" size="1em" request-event-name="remove-from-wish-list" course="${data.parentkey ? `${data.parentkey}${data.centerid ? `_${data.centerid}` : ''}` : `${data.kurs_typ}_${data.kurs_id}_${data.centerid}`}"></a-icon-mdx>` : ''}
```
Comment (Weedshaker) [inline src/es/components/molecules/tile/Tile.js:352]: a getter for hasWishlistTrash or so could resolve some redundancy with sensitive logic

File: src/es/components/molecules/tile/Tile.js [hunk 3]
```diff
@@ -369,7 +369,7 @@ export default class Tile extends Shadow() {
       <div class="m-tile__foot-passed">
         <span class="m-tile__passed-message">${this.getAttribute("passed-message")}</span>
         <div class="m-tile__foot-left">
-          ${this.hasAttribute('is-wish-list') ? /* html */`<a-icon-mdx namespace="icon-mdx-ks-" icon-name="Trash" size="1em" request-event-name="remove-from-wish-list" course="${data.parentkey ? data.parentkey : `${data.kurs_typ}_${data.kurs_id}_${data.centerid}`}"></a-icon-mdx>` : ''}
+          ${this.hasAttribute('is-wish-list') && !this.hasAttribute('is-info-events') ? /* html */`<a-icon-mdx namespace="icon-mdx-ks-" icon-name="Trash" size="1em" request-event-name="remove-from-wish-list" course="${data.parentkey ? data.parentkey : `${data.kurs_typ}_${data.kurs_id}_${data.centerid}`}"></a-icon-mdx>` : ''}
```
Comment (Weedshaker) [inline src/es/components/molecules/tile/Tile.js:372]: redundant

**Review by Weedshaker — APPROVED** [review]
solid job 👍

**Review by Weedshaker — DISMISSED** [review]

**Pull request merged**